src/components/flat-table/flat-table.pw.tsx

```diff
@@ -0,0 +1,3619 @@
+      "width",
+      "50px"
+    );
+  });
+
+  test(`should render cell with set width`, async ({ mount, page }) => {
+    await mount(<FlatTableTitleAlignComponent />);
+
+    await expect(flatTableCell(page, 0).locator("div")).toHaveCSS(
+      "width",
+      "60px"
+    );
+  });
+
+  test(`should render row header with set align`, async ({ mount, page }) => {
+    await mount(<FlatTableTitleAlignComponent />);
+
+    await expect(flatTableRowHeader(page).nth(1).locator("div")).toHaveCSS(
+      "text-align",
+      "left"
+    );
+    await expect(flatTableRowHeader(page).nth(2).locator("div")).toHaveCSS(
+      "text-align",
+      "center"
+    );
+    await expect(flatTableRowHeader(page).nth(3).locator("div")).toHaveCSS(
+      "text-align",
+      "right"
+    );
+    await expect(flatTableRowHeader(page).nth(4).locator("div")).toHaveCSS(
+      "text-align",
+      "left"
+    );
+  });
+
+  test(`should render cell with set align`, async ({ mount, page }) => {
+    await mount(<FlatTableTitleAlignComponent />);
+
+    await expect(
+      flatTableBodyRowByPosition(page, 0).locator("td").nth(0)
+    ).toHaveCSS("text-align", "left");
+    await expect(
+      flatTableBodyRowByPosition(page, 1).locator("td").nth(0)
+    ).toHaveCSS("text-align", "center");
+    await expect(
+      flatTableBodyRowByPosition(page, 2).locator("td").nth(0)
+    ).toHaveCSS("text-align", "right");
+    await expect(
+      flatTableBodyRowByPosition(page, 3).locator("td").nth(0)
+    ).toHaveCSS("text-align", "left");
+  });
+
+  ([
+    [0, "8px"],
+    [1, "16px"],
+    [2, "24px"],
+    [3, "32px"],
+  ] as const).forEach(([row, customPad]) => {
+    test(`should render row ${row} with ${customPad} custom padding`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<FlatTableCustomPaddingComponent />);
+
+      const rowByPosition0 = flatTableBodyRowByPosition(page, row)
+        .locator("td")
+        .nth(0)
+        .locator("div");
+      const rowByPosition1 = flatTableBodyRowByPosition(page, row)
+        .locator("td")
+        .nth(1)
+        .locator("div");
+      const rowByPosition2 = flatTableBodyRowByPosition(page, row)
+        .locator("td")
+        .nth(2)
+        .locator("div");
+      const rowByPosition3 = flatTableBodyRowByPosition(page, row)
+        .locator("td")
+        .nth(3)
+        .locator("div");
+
+      await expect(rowByPosition0).toHaveCSS("padding-left", customPad);
+      await expect(rowByPosition0).toHaveCSS("padding-right", customPad);
+      await expect(rowByPosition1).toHaveCSS("padding-left", customPad);
+      await expect(rowByPosition1).toHaveCSS("padding-right", "16px");
+      await expect(rowByPosition2).toHaveCSS("padding", customPad);
+      await expect(rowByPosition3).toHaveCSS("padding-left", customPad);
+      await expect(rowByPosition3).toHaveCSS("padding-right", "16px");
+    });
+  });
+
+  ([
```

Comment:
suggestion: this array is used in several tests so may be worth storing in a variable that way it only needs updating once if we want to change anything etc